Example.Avalonia/Example.Avalonia.csproj

```diff
@@ -27,11 +27,11 @@
   <ItemGroup>
     <PackageReference Include="Avalonia.Desktop" Version="0.10.0-preview6" />
     <PackageReference Include="NLog" Version="4.6.2" />
-    <PackageReference Include="ViewGenerator" Version="1.0.307">
+    <PackageReference Include="ViewGenerator" Version="1.0.308">
```

Comment:
You will break the build if you just increment the example's version right away

Reply:
Yeah, I incremented ViewGenerator/ViewGeneratorCore on Friday. These versions are already on Azure ;)

Reply:
👍